fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/facades/SimulationFacade.java

```diff
@@ -384,7 +384,13 @@ public TraciSimulationStepResult simulateUntil(long time) throws InternalFederat
                     vehicleData = new VehicleData.Builder(time, lastVehicleData.getName())
                             .copyFrom(lastVehicleData)
```

Comment:
we do not need `copyFrom` call here anymore, do we?

Reply:
I'm not sure. Wouldn't some values be `null`, if we don't call `copyFrom()`?

Reply:
hm.. i guess we should better make sure that no values are copied and set them explicitly. I also would set the emission/consumption objects with value 0. Currently you copy them from the last step which could lead to current consumption/emission values > 0 during parking.